### PR TITLE
[CT-1101] 024_custom_schema_tests

### LINKED ISSUE #5732: [CT-1101] 024_custom_schema_tests

### PR DESCRIPTION
**Pull request by colin-rogers-dbt**
resolves #5732

### Checklist

- [x] I have read [the contributing guide](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md) and understand what's expected of me
- [x] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
- [x] I have run this code in development and it appears to resolve the stated issue
- [x] This PR includes tests, or tests are not required/relevant for this PR
- [x] I have [opened an issue to add/update docs](https://github.com/dbt-labs/docs.getdbt.com/issues/new/choose), or docs changes are not required/relevant for this PR
- [x] I have run `changie new` to [create a changelog entry](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#Adding-CHANGELOG-Entry)

### CODE REVIEW

**Comment by McKnight-42** [comment]
@colin-rogers-dbt this is looking fantastic! a few small things I saw are that for test conversions we typically actually skip the changelog entry (as we see it as just a adjustment of current functionality), and that there isn't currently a PR for the [dbt-snowflake](https://github.com/dbt-labs/dbt-snowflake/tree/main/tests/integration/custom_schema_test) version of this test.

File: tests/functional/schema/test_custom_schema.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+            f"{_TABLE_TWO_DOT_MODEL_NAME}.sql": _TABLE_TWO_DOT_MODEL,
+            "table_3.sql": _TABLE_THREE_DOT_MODEL,
+        }
+
+    def test__postgres__custom_schema_from_model_name(
+        self, project, macros, project_config_update
+    ):
+        project.run_sql(_VALIDATION_SQL)
+        run_dbt(["seed"])
+        # run_dbt(["build"])
```
Comment (iknox-fa) [inline tests/functional/schema/test_custom_schema.py:203]: Any reason this was left in as a comment?

Comment (colin-rogers-dbt) [reply tests/functional/schema/test_custom_schema.py:203]: mostly cause I forgot to delete it

**Comment by colin-rogers-dbt** [comment]
> @colin-rogers-dbt this is looking fantastic! a few small things I saw are that for test conversions we typically actually skip the changelog entry (as we see it as just a adjustment of current functionality)

makes sense, should I just delete the entry? 

> there isn't currently a PR for the [dbt-snowflake](https://github.com/dbt-labs/dbt-snowflake/tree/main/tests/integration/custom_schema_test) version of this test.

Will raise a pr there to delete/migrate the tests in dbt-snowflake, do I need to check the other adapters as well?

**Review by iknox-fa — APPROVED** [review]
One errant comment, otherwise LGTM

**Comment by McKnight-42** [comment]
> 

yes should just be able to delete the entry then apply the skip changelog label to your pr so it skips that check

and I believe snowflake is the only other adapter to have a modified version of this test but yeah double checking is always good! 👍

**Pull request merged**